apps/extension/.env

```diff
@@ -1,6 +1,6 @@
 
 PRAX=lkpmkhpnhknhmibgnmmhdhgdilepfghe
-IDB_VERSION=31
+IDB_VERSION=32
```

Comment:
Anyone who has duplicate epochs in their database already will need to regenerate their data from scratch — hence the version bump.